KoalaBot.py

```diff
@@ -60,18 +62,15 @@ def get_config_from_argv():
             config_dir += "/"
 
         if os.name == 'nt' and config_dir[1] != ":":
-            if config_dir[0] == "/":
+            if config_dir[0] == "/" and os.getcwd()[-1] == "/":

```

Comment:
is it worth using the pathlib module for this kind of thing? https://docs.python.org/3/library/pathlib.html